src/Http/Routing.Abstractions/src/IRouter.cs

```diff
@@ -5,10 +5,22 @@
 
 namespace Microsoft.AspNetCore.Routing
 {
+    /// <summary>
+    ///
```

Comment:
?

Reply:
Fixed in https://github.com/dotnet/aspnetcore/pull/26513